src/structurizr/model/model.py

```diff
@@ -132,6 +127,15 @@ def people(self) -> Set[Person]:
         """Return the people in the model."""
         return {e for e in self.get_elements() if isinstance(e, Person)}
 
+    @property
+    def deployment_nodes(self) -> Set[DeploymentNode]:
+        """Return the *top level* deployment nodes in the model."""
+        return {
+            e
+            for e in self.get_elements()
+            if isinstance(e, DeploymentNode) and e.parent is None
```

Comment:
Flake8 will complain about this:

```suggestion
            element
            for element in self.get_elements()
            if isinstance(element, DeploymentNode) and element.parent is None
```